app/src/main/res/values/strings.xml

```diff
@@ -5,108 +5,108 @@
-    <string name="analytics_description">Allow Amarok to send logs for analytics.</string>
-    <string name="about_description">Show about.</string>
+    <string name="show_about">Show \"About\"</string>
+    <string name="moto">Hunting alone in the starless night.</string>
+    <string name="enable_analytics">Turn on sending logs</string>
+    <string name="analytics_description">Allow Amarok to send logs to the developer for analysis.</string>
+    <string name="about_description">Shows info about the app.</string>
     <string name="usage">Usage</string>
-    <string name="usage_description">Open usage.</string>
-    <string name="switch_app_hider">Switch AppHider Workmode</string>
-    <string name="current_mode">Current Mode: %s</string>
+    <string name="usage_description">Open info about usage.</string>
+    <string name="switch_app_hider">Switch app-hiding mode</string>
+    <string name="current_mode">Currently: %s</string>
     <string name="more_settings">More Settings</string>
-    <string name="switch_file_hider">Switch FileHider Workmode</string>
-    <string name="apply_on_restart">Changes will be applied on next restart.</string>
+    <string name="switch_file_hider">Switch file-hiding mode</string>
+    <string name="apply_on_restart">Restart the app to use the new changes.</string>
     <string name="hidden_toast">Hidden. Cock-a-doodle-do~!</string>
     <string name="unhidden_toast">Unhidden. Be careful.</string>
-    <string name="show_debug_info">Debug Info</string>
-    <string name="debug_info_description">Show debug info</string>
-    <string name="invalid_dsm_provider">DSM: Invalid DSM Provider. Have you installed and activated the DSM Provider on your device?</string>
-    <string name="hide_app_failed_admin_inactive">Hide App failed: Admin not active.</string>
-    <string name="add_hide_files">Add Hide Folder</string>
-    <string name="folder_selector_hints">Hints for folder selector: \n\t 1. Click \'Use This Folder\' to select the Folder\n\t 2. Always select the file from the local storage \n\t(You can slide from the right of the screen to open local files)</string>
-    <string name="remove_hide_path">Remove Hide Path?</string>
-    <string name="remove_hide_path_description">Are you sure to remove path \'%s\' from hiding?</string>
+    <string name="show_debug_info">Debug info</string>
+    <string name="debug_info_description">Shows info to help fix problems.</string>
+    <string name="invalid_dsm_provider">DSM: Invalid DSM provider. Have you installed and activated the DSM provider on your device?</string>
+    <string name="hide_app_failed_admin_inactive">Could not hide app: Administration not active.</string>
+    <string name="add_hide_files">Add hidden folder</string>
+    <string name="folder_selector_hints">Hints for folder selector: \n\t 1. Click \"Use This Folder\" to select the Folder\n\t 2. Always select a file on the device \n\t(Slide from the right of the screen to open local files)</string>
```

Comment:
There is no "Use This Folder" string…

Reply:
Ah, It's a button in the android filepicker provided by the system:

<img src="https://user-images.githubusercontent.com/41465688/217402646-9b9cc5cf-c72a-4642-b7b9-3d56737e55b6.png" height="150">